Makefile

```diff
@@ -56,29 +59,41 @@ TOC_END=<\!--te-->
 TOC_PATH=toc.md
 
 all: prepare test build
-.PHONY: all download test test-ci build install build-mac build-linux build-windows build-all coverage clean ramdisk rest-server nightly toc generate-install syslog checkdoc
+.PHONY: test test-ci coverage
+.PHONY: download download-restic-key
+.PHONY: build build-mac build-linux build-pi build-windows build-no-selfupdate build-all
+.PHONY: generate-config-reference generate-jsonschema generate-install generate-restic
+.PHONY: all verify prepare install clean ramdisk rest-server nightly toc syslog checkdoc
+
+verify:
+ifeq ($(wildcard $(GOPATH)/.),)
+	@echo "GOPATH not found, please check your go installation"
+	exit 1
+endif
 
-$(GOBIN)/eget:
+$(GOBIN)/eget: verify
 	@echo "[*] $@"
-	go install -v github.com/zyedidia/eget@latest
+	GOBIN="$(GOBIN)" \
+	$(GOCMD) install -v github.com/zyedidia/eget@latest
 
-$(GOBIN)/goreleaser: $(GOBIN)/eget
+$(GOBIN)/goreleaser: verify $(GOBIN)/eget
 	@echo "[*] $@"
-	eget goreleaser/goreleaser --upgrade-only --to $(GOBIN)
+	"$(GOBIN)/eget" goreleaser/goreleaser --upgrade-only --to $(GOBIN)
 
-$(GOBIN)/github-markdown-toc.go: $(GOBIN)/eget
+$(GOBIN)/github-markdown-toc.go: verify $(GOBIN)/eget
```

Comment:
this was quite handy for a while but I don't think we need this tool anymore 😉

Reply:
Didn’t check for orphan steps though in this PR.

Reply:
Don't worry about it, I'll clean it up some time